.eslintrc.js

```diff
@@ -140,7 +144,11 @@ module.exports = {
     'local/no-es2015-number-props': 2,
     'local/no-export-side-effect': 2,
     'local/no-for-of-statement': 2,
-    'local/no-forbidden-terms': 2,
+    'local/no-forbidden-terms': [
+      2,
+      forbiddenTermsGlobal,
+      forbiddenTermsSrcInclusive,
+    ],
```

Comment:
On a previous PR, it was mentioned that it was odd that `.estlintrc.js` imported a function into configuration.

This is similar, but I think fundamentally different:

- This imports a configuration constant, so it makes sense as configuration.

- eslint plugins (at least on VSCode) will reload the configuration automatically, but not the rule implementation. By moving the terms into `.eslintrc.js`, files will be updated when you, say, update an `allowlist`. Otherwise, you'd have to reload the entire editor.

Reply:
Excellent rationale. Can't disagree :)